blueocean-dashboard/src/main/js/creation/github/steps/GithubRepositoryStep.jsx

```diff
@@ -31,9 +42,14 @@ export default class GithubRepositoryStep extends React.Component {
         const orgName = flowManager.selectedOrganization.name;
         const existingPipelineCount = flowManager.existingPipelineCount;
         const sortedRepos = flowManager.selectableRepositories.slice().sort(this._sortRepos);
+        const loading = flowManager.repositoriesLoading;
 
         return (
-            <FlowStep {...this.props} className="github-repo-list-step" title={title} disabled={disabled}>
+            <FlowStep {...this.props} className="github-repo-list-step" title={title} loading={loading} disabled={disabled}>
+                <div className="loading-msg">
+                    { this._getLoadingMessage()}
```

Comment:
guessing this is part of the design? seems like a weird way to show loading status of something that appears below it in the page... I'd just have an infinite scroll list or something, but that's me.